lib/takeaway.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'menu'
+require_relative 'order'
+
+class Takeaway
+    attr_reader :menu, :order, :food_list
+    
+    def initialize(menu= Menu.new)
+        @menu = menu
+        @total_price = 0
+        @food_list = []
+        
+    end
+
+    def view
+        @menu.view
+    end
+
+    def pick(food, number)
+        new_order
+        number.times { food_list.push(food => @menu.menu[food])}
+        @food_list
+    end
+
+    def basket
+      @amount = @order.sum(@food_list) 
+      @amount
+      
+     end
+
+    def new_order(order= Order.new)
```

Comment:
Is `new_order` supposed to be used directly by the user, or should it be private?